docs/ways-of-working.md

```diff
@@ -41,8 +41,18 @@ As mentioned before, any significant changes in a PR should be accompanied with
 Smaller changes should be self documented in the tests. With this approach, any large feature or change has user friendly documentation, whereas technical or implementation details are documented for developers by the tests.
 
 ## Coding style
+### Java
 Please ensure your coding style is consistent with the rest of the Gaffer project and the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html). Your changes should pass the checkstyle and spotless plugins that are part of the continuous integration pipeline and check for code formatting and licenses. Before you push your changes you can check the checkstyle plugin passes with `mvn checkstyle:check` and check the spotless plugin passes with `mvn spotless:check`.
 
+### Python
+Please ensure your coding style is consistent with the rest of the Gaffer project and the [PEP 8 Style Guide](https://peps.python.org/pep-0008/). However, there are a few exceptions to the standards set by PEP8:
+* Module level imports at the top of the file -  this will not be enforce but is recommended where it does not cause issues with the code generated by Fishbowl.
+* Max line length of 79 characters - the max line length that will be enforced in this project has been increased to 100 characters.
+
+Before you create a PR for your changes you can use [autopep8](https://github.com/hhatto/autopep8) to check and fix any styling issues. The following can be run which will take into account the rule exceptions mentioned above.
+`autopep8 --exit-code --r --i --a --a --max-line-length 100 --ignore E402 .`
```

Comment:
These should only have one dash, see the [pipeline](https://github.com/gchq/gaffer-tools/blob/2af6914e65a6987b0a32a61a3d27ed52917c0836/.github/workflows/continuous-integration.yaml#L21)